app/api/v1/endpoints/projections.py

```
@@ -0,0 +1,202 @@
+)
+from app.schemas.projection import (
+    ProjectionDefinitionCreateRequest,
+    ProjectionDefinitionResponse,
+    ProjectionStateListItem,
+    ProjectionStateResponse,
+)
+
+router = APIRouter()
+
+
+def _definition_to_response(r: Any) -> ProjectionDefinitionResponse:
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `r`

(ANN401)